pep-0581.rst

```diff
@@ -0,0 +1,362 @@
+
+Thanks to Ernest W. Durbin III for working on this.
+
+Update the CLA host
+-------------------
+
+At the moment the CLA is hosted within bpo. It needs to be updated such that
+signing the CLA does not require a bpo account, and it should be hosted outside
+of the bpo.
+
+The current CLA process itself is not ideal. Currently contributors to
+devguide, peps, and core-workflow need to sign a CLA,  and it requires a bpo
+account. A bpo account should not be required for those projects.
+
+Currently the CLA process involves real human to manually check the records.
+Question: Will it be possible to completely automate the CLA process, so
+it does not require human intervention?
+
+Create labels for issue triage
+------------------------------
+
+In bpo, we currently have the following fields for each issues:
+
+Type: behavior, crash, compile error, resource usage, security, performance, enhancement.
+Components: 2to3, Argument Clinic, asyncio, Build, Cross-build, ctypes, ...
+Priority: release blocker, deferred blocker, critical, high, normal, low
+
+We will create the corresponding labels::
+
+   type-behavior, type-crash, type-compile error, type-resource usage, ...
+
+   components-2to3, components-argument clinic, components-asyncio, ...
+
+   priority-release blocker, priority-deferred blocker, priority-critical, ...
+
+In addition, we'll create ``needs triage`` label.
+
+Create issue template
+---------------------
+
+We will create an issue template and add the following headers::
+
+   ---
+   Type: behavior | crash | compile error | resource usage (choose one)
+   Components: 2to3 | Argument Clinic | asyncio | Build | ... (can select more than one)
+   Priority: release blocker | deferred blocker | critical | ...
+   Needs backport to: 2.7 | 3.6 | 3.7
+   ---
+
+The idea is to allow the issue creator to help us triage the issue.
+The above values are pre-filled in the template. Issue creator will remove texts
+that do not apply to their issue.
+
+Based on the above headers, bedevere-bot can apply the necessary labels to the
+issue. If issue creator did not supply the above headers, the bot will apply
+``needs triage`` label. At that point it will require a core developer to
+properly label the issue.
+
+Updates to bedevere
+-------------------
+
+Bedevere-bot will need to be updated to recognize the issue headers described above,
+and apply the proper labels.
+
+Bedevere-bot can also copy over the labels to pull requests that correspond to
+the issue.
+
+Update the devguide
+-------------------
+
+Provide explanation in the devguide about new issue workflow and triage labels.
+
+Add a button in bpo to migrate the issue to GitHub
+--------------------------------------------------
+
+This will require actual update to the bpo. But I believe this effort needed
+is much less than a complete overhaul.
+
+We will create a button in bpo, that will copy over the issue description
+and associated comments into a GitHub issue.
```

Comment:
What about the existing files, which are typically patch files?  Create a comment to embed them in? Or put a link to the original in?